cpp/src/ray/config_internal.cc

```diff
@@ -118,12 +118,14 @@ void ConfigInternal::Init(RayConfig &config, int argc, char **argv) {
     startup_token = absl::GetFlag<int64_t>(FLAGS_startup_token);
   }
   if (worker_type == WorkerType::DRIVER && run_mode == RunMode::CLUSTER) {
-    if (redis_ip.empty()) {
+    if ((::RayConfig::instance().bootstrap_with_gcs() && gcs_ip.empty()) ||
```

Comment:
Is the leading `::` necessary?

Reply:
Yes. In ray cpp, there is another RayConfig unfortunately.